NDTensors/ext/NDTensorsCUDAExt/copyto.jl

```diff
@@ -0,0 +1,6 @@
+# Catches a bug in `copyto!` in CuArray backend.
+function Base.copyto!(
+  dest::Exposed{<:CuArray}, src::Exposed{<:CuArray,<:Base.ReshapedArray}
+)
+  return copyto!(dest, expose(parent(src)))
```

Comment:
Same thing, please return `unexpose(dest)`.

Reply:
Okay I will change this in the NDTensorMetal code too

Reply:
I have updated the code!